Makefile

```diff
@@ -81,3 +85,18 @@ pip-requirements-test: venv
 		pip install -e ".[test]" > /dev/null
 
 .PHONY: dist
+else
+PYTHON_VERSION = 3.9
```

Comment:
We should take that out of the conditional, even though it's only used here, I think that makes it more discoverable.